docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+
+services:
+
+  wildfly:
+    build: .
+    container_name: "wildfly"
+    ports:
+      - "8080:8080"
+      - "9990:9990"
+
+  postgres:
+    image: postgres:16.2
+    container_name: "postgres"
+    environment:
+      - "POSTGRES_PASSWORD=postgres"
```

Comment:
@BarbieQkiller Could this pose a security risk? Would it maybe be good to use .env file and add that in document as a step for setting up this application? So in the .env file it could look something like this:

`POSTGRES_PASSWORD=my_secure_password`

and in the .yml file:

`- "POSTGRES_PASSWORD=${POSTGRES_PASSWORD}"` instead of `- "POSTGRES_PASSWORD=postgres"`

What do you think? 🤔

Reply:
We could add an .env file later as an improvement. I don't think we have to worry about security at this stage in the development.

Reply:
Okay lets do that. Should I make issue as reminder and add "reminder" label? Or is that overkill?

Reply:
Great suggestion! That way we can decide whether we want to implement it and when. ☺️